code/modules/projectiles/projectile/ego_bullets/waw.dm

```diff
@@ -167,3 +150,29 @@
 	damage = 145
 	damage_type = RED_DAMAGE
 	flag = RED_DAMAGE
+
+//Assonance, our one hitscan laser
+/obj/projectile/beam/assonance
+	name = "laser tag beam"
```

Comment:
Nice name.

Reply:
It still needs the white laser Sprite from the yang PR, will fix the name at least